ivy/container/elementwise.py

```diff
@@ -6473,3 +6473,95 @@ def minimum(
 
         """
         return self.static_minimum(self, x2, key_chains, to_apply, prune_unapplied, map_sequences, out=out)
+
+
+
+    @staticmethod
+    def static_maximum(
+        x1: Union[ivy.Container, ivy.Array, ivy.NativeArray],
+        x2: Union[ivy.Container, ivy.Array, ivy.NativeArray],
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
+        *,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        """Computes the elementwise maximum between this container and another container
```

Comment:
these docstrings are wrong, please re-read the **classes** section [here](https://github.com/unifyai/ivy/blob/e7b3211dbe1d24ff651155d7899546a7c4dc30f7/docs/partial_source/deep_dive/12_docstrings.rst)